packages/router/src/__typetests__/routeParamsTypes.test.ts

```diff
@@ -70,33 +70,33 @@ describe('RouteParams<>', () => {
       toDate: '2021/11/17',
     }
 
-    expectType<string>(globRoutes.fromDate)
-    expectType<string>(globRoutes.toDate)
+    expect(globRoutes.fromDate).type.toBeString()
+    expect(globRoutes.toDate).type.toBeString()
   })
 
   test('Single Glob route params', () => {
     const globRoutes: RouteParams<'/from/{fromDate...}'> = {
       fromDate: '2021/11/03',
     }
 
-    expectType<string>(globRoutes.fromDate)
+    expect(globRoutes.fromDate).type.toBeString()
   })
 
   test('Starts with Glob route params', () => {
     const globRoutes: RouteParams<'/{description...}-little/kittens'> = {
       description: 'cute',
     }
 
-    expectType<string>(globRoutes.description)
+    expect(globRoutes.description).type.toBeString()
   })
 
-  test('Glob params in the middle', () => {
+  context('Glob params in the middle', () => {
```

Comment:
This `test()` is nested inside other `test()`. Currently this is reported as an error, but TSTyche will allow subtest style soon. Only that I am not sure if you want to mix styles.

Reply:
Yeah probably just a result of copy pasting :). Context is the way to nest it inside the test block I guess?

Reply:
This `context()` is an alias of `describe()`. It is nested inside another `describe()`.

If `context()` looks too exotic in this context, we can simply use a `describe()`. It does the same job.

Reply:
To be honest, nested `test()` looks better than `context()`. Somehow more clear. `context()` is too abstract, because everything can be some context. It does not sound anyhow specific. `test()` makes it clear that something is being tested. I wanted to have `context()` in the API to allow nesting. Its name was not convincing, but I could not find a better alternative. But now I see one ;D

Reply:
Yeah maybe just `it` could work!